website/docs/d/datasource_compute_image.html.markdown

```diff
@@ -0,0 +1,81 @@
+---
+layout: "google"
+page_title: "Google: google_compute_image"
+sidebar_current: "docs-google-datasource-compute-image"
+description: |-
+  Get information about a Google Compute Image.
+---
+
+# google\_compute\_image
+
+Get information about a Google Compute Image. Check that your service account has the `compute.imageUser` role if you want to share [custom images](https://cloud.google.com/compute/docs/images/sharing-images-across-projects) from another project. If you want to use [public images](https://cloud.google.com/compute/docs/images#os-compute-support), do not forget to specify the dedicated project. For more information see
+[the official documentation](https://cloud.google.com/compute/docs/images) and its [API](https://cloud.google.com/compute/docs/reference/latest/images).
+
+## Example Usage
+
+```hcl
+data "google_compute_image" "my_image" {
+  name = "image-family"
```

Comment:
This seems misleading. Let's either do `family = "image-family"` or `name = "image-name"`